wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -189,6 +189,9 @@ public Wasp build() {
         }
 
         private void init() {
+            if (endPointUrl == null) {
```

Comment:
This is a requirement to setup the library, if something is missing, it should throw the exception. There is nothing wrong with this. Adding every exception to the method signature is bad approach. If you don't throw the exception, it will mislead everything. Every library is actually like this and should throw exception when something is wrong. It is developer responsibility